src/ingest_validation_tools/error_report.py

```diff
@@ -85,19 +86,63 @@ def dict_type_counts(self):
                     )
         return counts_by_sub_category
 
+    @property
+    def cleaned_value(self):
```

Comment:
Deprecated message_munger.py (except for legacy integration into table_validator.py).

Replicated useful functionality (cleanup quotes, add trailing punctuation), can be turned on or off by fields in ErrorDict.